src/Connections/GitlabRepo.ts

```diff
@@ -642,13 +655,17 @@ ${data.description}`;
             if (commentNotes) {
                 existing.commentNotes = [...(existing.commentNotes ?? []), ...commentNotes];
             }
-            existing.commentCount = opts.commentCount;
+            existing.commentCount = existing.commentCount + opts.commentCount;
```

Comment:
This suggestion is just fancy syntax sugar. No need to apply.
```suggestion
            existing.commentCount += opts.commentCount;
```